server/views/components/instagram-thumbnail/index.njk

```diff
@@ -0,0 +1,31 @@
+<a href="{{ model.url }}" class="instagram-thumbnail"
+  data-track-click='{"screenName": "{{ model.screenName }}", "createdAt": "{{ model.createdAt }}", "postId": {{ model.id }}}'>
```

Comment:
`screenName` could be a retweet. The tweet's creation date seemed like quite a useful bit of information to hold on to to me, but I won't fight too hard for it to stay.